### PR TITLE
Docstring Python escaping backslash in latex strings sdded

### PR DESCRIPTION
**Pull request by gxyd**
To see why i am doing this, see my [question here](https://gitter.im/sympy/sympy?at=560a1c54519547fc1e3aef0b), and [this response](https://gitter.im/sympy/sympy?at=560abc04ff22c70f6fab2e7d)

### CODE REVIEW

**Comment by gxyd** [comment]
@asmeurer please review.

**Comment by gxyd** [comment]
ping @leosartaj .

File: sympy/printing/latex.py [hunk 1]
```diff
@@ -1907,6 +1907,12 @@ def latex(expr, **settings):
     >>> print(latex((2*tau)**Rational(7,2)))
     8 \sqrt{2} \tau^{\frac{7}{2}}
 
+    Not using a print statement for printing, results in double slashes for
```
Comment (leosartaj) [inline sympy/printing/latex.py]: double backslashes?

**Comment by leosartaj** [comment]
+1, to merge.

**Comment by leosartaj** [comment]
Thanks @gxyd

**Comment by gxyd** [comment]
Thanks @leosartaj

**Pull request merged**